### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by rejamer** [opencrx/opencrx#132] [Q&A]
I just discovered openCRX recently and successfully installed with no issues. I am having trouble getting familiar with how basic processes work. I found a getting started guide and users guide but neither were very helpful. After days of struggling, I hve to move on or figure out the basics - I have used many CRM packages (eg Saleslogix, etc.) but struggling here. For example, I figured out that an account can be anything: person, legal entity, etc. What I cannot figure out is simpy how to associate a contact with a company - seems like basic stuff and more than a little frustrating ... any doc I am overlooking? Suggestions?

**Comment by wfro64** [comment]
An account can be a person (= Contact), legal entity, group or unspecified (see https://www.opencrx.org/opencrx/5.3/uml/opencrx-core/account1/tn/010_Main.png.html). Accounts are linked by creating member relationsships (see https://www.opencrx.org/opencrx/5.3/uml/opencrx-core/account1/tn/060_Membership.png.html)

A contact and a company are linked as follows:
* create the contact
* create the company
* create the member relationship

# Create the contact
Navigate to Manage Accounts > Accounts > New Contact

![Screenshot_20231030_084035](https://github.com/opencrx/opencrx/assets/11387061/5845a3ac-6abc-43be-9417-db87c1f7c098)

![Screenshot_20231030_084301](https://github.com/opencrx/opencrx/assets/11387061/44aeab8b-66cf-4ecb-8856-dab16402ecc8)

# Create the company
Navigate to Manage Accounts > Accounts > New Legal Entity 

![Screenshot_20231030_084111](https://github.com/opencrx/opencrx/assets/11387061/45c0f25c-edea-4341-b736-0ff0ed605c7f)

# Create member relationship
On the newly created company navigate to Members > New > Member

![Screenshot_20231030_084413](https://github.com/opencrx/opencrx/assets/11387061/edb83eaf-a235-44ba-a960-ba5fe6adc6f0)

![Screenshot_20231030_084451](https://github.com/opencrx/opencrx/assets/11387061/2ea6fc61-6ff3-49a8-8032-d16305b302a5)

![Screenshot_20231030_084517](https://github.com/opencrx/opencrx/assets/11387061/aa0d794b-eb69-48b4-a2d0-93f01312d073)

NOTE:
The standard UI is generic and rather intended for advanced users. Due to the open interfaces, openCRX supports the development of own, use-case specific UIs.
